src/keycloak/chart/values.yaml

```diff
@@ -339,6 +339,9 @@ autoscaling:
   # The scaling policy to use. This will scale up quickly but only scale down a single Pod per 5 minutes.
   # This is important because caches are usually only replicated to 2 Pods and if one of those Pods is terminated this will give the cluster time to recover.
   behavior:
+    scaleUp:
+      # Keycloak CPU spikes at boot time, this aims to prevent massive unwanted scale outs during the boot time or upgrades
+      stabilizationWindowSeconds: 600
```

Comment:
This is probably the most crucial part to avoid HPA misbehaving. 

Keycloak uses extensive CPU when booting up and rebalancing the cluster. Booting a cluster of 3 takes roughly 5 minutes and can lead to scale out events (specifically at AKS where for some reason the CPU spikes are higher). Booting up 5 nodes usually causes Zarf to timeout.